src/ceph-detect-init/Makefile.am

```diff
@@ -55,11 +55,13 @@ EXTRA_DIST += \
 
 ceph-detect-init-all: ceph-detect-init/virtualenv
 
+CEPH_DETECT_INIT_VIRTUALENV := .
```

Comment:
the proposed solution would allow a per-module basis:

    ${CEPH_BUILD_VIRTUALENVS}/ceph-detect-init/
    ${CEPH_BUILD_VIRTUALENVS}/ceph-disk/

The only thing a build system would need to define is the parent virtualenv dir